### PR TITLE
Replace ant with gradle

### PR DESCRIPTION
**Pull request by egor-bukharov**
Build system changed from Ant to Gradle.
Implemented new integration test.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 bin
-screenshotTaker.jar
\ No newline at end of file
+screenshotTaker.jar
+adbScreenshotTest.png
+screenshotTakerTest.png
```
Comment (jmoody) [inline .gitignore]: Add these to .gitignore

```
build
.gradle
```

File: gradlew [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+# Collect all arguments for the java command, following the shell quoting and substitution rules
+eval set -- $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS "\"-Dorg.gradle.appname=$APP_BASE_NAME\"" -classpath "\"$CLASSPATH\"" org.gradle.wrapper.GradleWrapperMain "$APP_ARGS"
+
+# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
+if [ "$(uname)" = "Darwin" ] && [ "$HOME" = "$PWD" ]; then
+  cd "$(dirname "$0")"
+fi
+
+exec "$JAVACMD" "$@"
```
Comment (jmoody) [inline gradlew]: Make this file executable:

```
$ chmod +x gradlew
```

File: .gitignore (hunk 1 above)
Comment (egor-bukharov) [reply .gitignore]: Yes, thank you!

**Comment by egor-bukharov** [comment]
@jmoody, `adb get-serialno` is used in `integrationTest` task to detect connected device, but it returns the same error code when no devices connected and when more than one device connected. Currently integrationTest task asks to provide value for ADB_SERIAL variable in both cases, but I will look how to improve this behaviour.

**Pull request merged**